### ISSUE TITLE
NVDA doesn't read headers

### ISSUE THREAD

**Issue by tomfisher12** [nvaccess/nvda#10871]
### Steps to reproduce:
1. Open provided jsFiddle
2. Run NVDA 
### Actual behavior:
Screen reader reads only anchor elements 
### Expected behavior:
Screen reader reads every element on a page
#### NVDA installed/portable/running from source:
https://www.nvda.pl/pobierz 
#### NVDA version:
2019.3.1
#### Windows version:
Windows 10 Home
#### Name and version of other software in use when reproducing the issue:
https://jsfiddle.net/6f21dvgo/
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No, this is the latest stable version

**Comment by lukaszgo1** [comment]
For me the headings are read. @tomfisher12 How exactly are you navigating this web page? Also in which browser?

**Comment by tomfisher12** [comment]
> For me the headings are read. @tomfisher12 How exactly are you navigating this web page? Also in which browser?

Navigate using tab, I use recent version of Google Chrome, there might be some css or scripts that can couse this issue?

**Comment by lukaszgo1** [comment]
Headings are not supposed to be present in the tab order, so the behavior that you are observing is a correct one. I vote for closing and marking as invalid.

**Comment by tomfisher12** [comment]
> Headings are not supposed to be present in the tab order, so the behavior that you are observing is a correct one. I vote for closing and marking as invalid.

How do I use tab to navigate to headings?

**Comment by lukaszgo1** [comment]
You do not. Headings are reachable with arrows when you are in browse mode

**Comment by feerrenrut** [comment]
I'm going to close this issue.

**Comment by Brian1Gaff** [comment]
You don't you use h I'd imagine.
 

bglists@blueyonder.co.uk 
Sent via blueyonder. 
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
